optimize/self-managed/optimize-deployment/migration-update/camunda-7/3.13-to-3.14.md

```diff
@@ -17,6 +17,11 @@ Here you will find information about:
 - Changes in behavior (for example, due to a new feature)
 - Changes in translation resources
 
+## Versioning
+
+From Optimize 3.14 onwards, instances of Optimize running with Camunda Platform 7 will exclusively be using the 3.x.x versioning scheme. Conversely, instances of Optimize running with Camunda Platform 8 will exclusively be using the 8.x.x versioning scheme. This means you will only be able to update to Optimize 3.14 if you are currently using Optimize 3.13 or below. Optimize instances of versions 8.5 and below will not be upgradeable to Optimize 3.14.
+In order to ensure that no Optimize 8 upgrade is executed on an Optimize instance previously running with Camunda Platform 7, the 3.14 upgrade runs a check against you database before executing the upgrade. This check detects whether any Camunda Platform 8 data is present in your setup and if so, the upgrade will be aborted to avoid erroneous upgrade paths. Specifically, it validates that there is no data present in the `position-based-import-index` which is exclusively used for Camunda Platform 8 data import. If you are encountering issues upgrading to 3.14 in your Camunda Platform 7 environment please contact Support.
```

Comment:
> please contact Support

Do all of our customers have the ability to do this? Or does it depend on their contract?

Reply:
I dont know what our contracts look like, but there are a few places in the docs where it mentions to contact support/"us"/customer success. @conceptualshark is there a guideline for this?

Reply:
@PHWaechtler I don't think we have any standards for this (please correct me if I'm wrong @christinaausley) but we absolutely should. My quick feeling for this would be that we use support (linked) or Customer Success Manager depending on the type of doc/issue and therefore where it should be directed. 

For example:

Contact [Camunda support](https://camunda.com/services/support/) if you encounter issues upgrading to 3.14 in your Camunda Platform 7 environment.

Contact your Customer Success Manager to increase your cluster size as this requires custom pricing.

NB: We always try to avoid using the term "please" in docs.